pint/models/spindown.py

```diff
@@ -25,29 +25,28 @@ def __init__(self):
         # The number of terms in the taylor exapansion of spin freq (F0...FN)
         self.num_spin_terms = maxderivs
 
```

Comment:
Agreed with all that was said.

If prefix parameters are not long double, then we can just define F0 separately, for the time being. (Yes, now I am being pragmatic)